src/main/java/de/cuuky/varo/command/varo/ColorcodesCommand.java

```diff
@@ -0,0 +1,35 @@
+package de.cuuky.varo.command.varo;
+
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+
+import de.cuuky.varo.Main;
+import de.cuuky.varo.command.VaroCommand;
+import de.cuuky.varo.player.VaroPlayer;
+
+public class ColorcodesCommand extends VaroCommand {
+
+	public ColorcodesCommand() {
+		super("colorcodes", "Zeigt alle Colorcodes an", "varo.colorcodes", "cc");
+	}
+
+	@Override
+	public void onCommand(CommandSender sender, VaroPlayer vp, Command cmd, String label, String[] args) {
+
+            sender.sendMessage("§c==============================");
```

Comment:
```suggestion
            sender.sendMessage("§7==============================");
```
Red is for error messages